sys/include/libufs.h

```diff
@@ -10,11 +10,22 @@
 
 
 struct Chan;
-struct mount;
 struct vnode;
+struct thread;
 
+/* Wrapper for a UFS mount.  Should support reading from both kernel and user
+ * space (eventually)
+ */
+struct mount {
+	struct ufsmount*	mnt_data;
```

Comment:
Ok - I'll look at a spatch to rename ufsmount in a later PR and make the typedef change for this one.